### ISSUE TITLE
Communities: See Community Rules

### ISSUE THREAD

**Issue by Samyoul** [status-im/status-react#17969]
# Feature Issue

[comment]: # (Please replace ... with your information. Remove < and >)
More details to follow.

### User Story

- As a user
    - When I join a community
- I want to see the community rules
- So I don’t get kicked out

![image](https://github.com/status-im/status-mobile/assets/5999878/4c1e5f8c-e829-43ff-88b3-9a6c7d19b3d1)

### Description

Summary: This feature focuses on ensuring that users are immediately aware of a community's rules upon joining. When a user joins a community, they should be presented with the community rules in a clear and accessible manner. This initiative is aimed at promoting awareness and understanding of the community guidelines, thereby reducing the likelihood of rule violations and potential removal from the community.

### Acceptance Criteria
- Follow Figma
- As soon as a user joins a community, the community rules should be prominently displayed.

### Technical tasks

- [x] Create a skeleton behind a toggle so that other devs can start working in parallel - https://github.com/status-im/status-mobile/pull/18030
- [x] Add community rules view, out of scope: selection of accounts, out of scope: slider if not implemented, a button is fine

### Questions

Do rules support markdown?

**Comment by ajayesivan** [comment]
We already have the bottom sheet with rules implemented. We don't show it for all communities, not sure what are the conditions it's based on.
<img width="300" src="https://github.com/status-im/status-mobile/assets/19279756/05de39d1-5849-4e94-87dd-7b9cc874b3ee"/>

**Comment by ajayesivan** [comment]
At the moment it's hardcoded: https://github.com/status-im/status-mobile/blob/6f9bcd1bb1754f7735d33e966717c82e32415c2f/src/status_im2/contexts/communities/actions/community_rules_list/view.cljs#L7-L29
